probe_scraper/check_repositories.py

```diff
@@ -0,0 +1,52 @@
+import os
+from pathlib import Path
+
+import requests as reqs
+import yaml
+from glean_parser.lint import lint_yaml_files
+
+GITHUB_RAW_URL = "https://raw.githubusercontent.com"
+REPOSITORIES = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "repositories.yaml"
+)
+validation_errors = []
+with open(REPOSITORIES) as data:
+    repos = yaml.load(data, Loader=yaml.FullLoader)
```

Comment:
If possible, can we use `SafeLoader` instead of `FullLoader`.  That will prevent loading YAML from running arbitrary code.